include/libtorrent/session_settings.hpp

```diff
@@ -1498,6 +1499,10 @@ namespace libtorrent
 		// 'ro' key (value = 1) in the top-level message dictionary of outgoing
 		// query messages.
 		bool read_only;
+
+		// the number of minutes a immutable/mutable item will be expired.
+		// default is 0, means never expires.
+		int item_lifetime;
```

Comment:
I believe all other timeouts in libtorrent are specified in seconds, and that it's probably a good idea to specify this one in seconds too, for consistency.